scripts/build-documentation.sh

```diff
@@ -12,3 +13,14 @@ make
 make docs
 
 echo "build Sphinx documentation FINISHED"
+
+
+COMMIT_ID="$(git rev-parse HEAD)"
+TEMP=$((git describe --exact-match $COMMIT_ID) 2>&1)
+
+if [[ $TEMP != *"fatal"* ]]; then
+    echo "New tag detected; Uploading the documentation to opae.github.io"
+    /bin/bash scripts/push-documentation.sh 
+else
+    echo "No new tag detected; Documentation will not be uploaded to opae.github.io"
+fi    
```

Comment:
I think travis has an environment variables for this type of information.
https://docs.travis-ci.com/user/environment-variables/
![image](https://user-images.githubusercontent.com/22808196/36797755-a12df8b6-1c5d-11e8-934f-8b9006fa88d4.png)
In particular, can you use the `TRAVIS_TAG` environment variable to determine if this is a tag?

Reply:
Needs some investigation to be done in order to figure out the usage. Can use git commands for now.

Reply:
Have you tested this? How?